examples/all_widgets.rs

```diff
@@ -179,7 +177,7 @@ fn draw_ui(gl: &mut Gl,
             .rgba(0.4, 0.75, 0.6, 1.0)
             .frame(demo.frame_width)
             .label("PRESS")
-            .callback(Box::new(|| demo.bg_color = Color::random()))
+            .callback(box |&mut:| demo.bg_color = Color::random())
```

Comment:
It complained about not being able to infer the closure's type otherwise? I've been getting this quite often, perhaps I'm just doing something wrong here?